client/web/src/components/diff/FileDiffNode.tsx

```diff
@@ -120,23 +120,25 @@ export const FileDiffNode: React.FunctionComponent<FileDiffNodeProps> = ({
                             </Badge>
                         )}
                         {stat}
-                        <Link to={{ ...location, hash: anchor }} className={classNames('ml-2', styles.headerPath)}>
-                            {path}
-                        </Link>
-                    </div>
-                    <div className={styles.headerActions}>
-                        {/* We only have a 'view' component for GitBlobs, but not for `VirtualFile`s. */}
-                        {node.mostRelevantFile.__typename === 'GitBlob' && (
-                            <Button
+                        {node.mostRelevantFile.__typename === 'GitBlob' ? (
+                            <Link
                                 to={node.mostRelevantFile.url}
                                 data-tooltip="View file at revision"
-                                variant="link"
-                                size="sm"
-                                as={Link}
+                                className="pr-0 pl-2 fw-bold"
                             >
-                                View
-                            </Button>
+                                <strong>{path}</strong>
+                            </Link>
+                        ) : (
+                            <span className="ml-2">{path}</span>
                         )}
+                        <Link
+                            to={{ ...location, hash: anchor }}
+                            className={classNames('ml-2', styles.headerPath)}
+                            data-tooltip="View diff"
+                            aria-label="View diff"
```

Comment:
```suggestion
                            data-tooltip="Pin diff"
                            aria-label="Pin diff"
```

or something similar? The user already looks at the diff :)